install/kubernetes/helm/istio/values.yaml

```diff
@@ -325,3 +325,7 @@ global:
   # Use the Mesh Control Protocol (MCP) for configuring Mixer and
   # Pilot. Requires galley (`--set galley.enabled=true`).
   useMCP: false
+
+  mixer:
+    # Setting this to true sets the useAdapaterCRDs mixer startup argument to false, effectively disabling UseAdapterCRDs.
+    disableUseAdapterCRDs: false
```

Comment:
This is a bit confusing as an argument.  What about just calling it `useAdapterCRDs` and having it default to `true`?  Is that not viable?  The meaning would be clearer, imho, at least.